docs/installation/others/creating-ds-instance.adoc

```diff
@@ -31,11 +31,11 @@ Customize the DS configuration file as follows:
 [literal,subs="+quotes,verbatim"]
```

Comment:
@LouiseMcGarry I think we are running into the same issue below where the '*' is being interpreted as 'bold'.  I tried to copy/paste the downstream blob under 6.4.2.1.1. Generating a DS configuration file, for example ds.inf and the '*'s disappeared.